include/aspect/stokes_matrix_free.h

```diff
@@ -530,6 +530,8 @@ namespace aspect
 
       Simulator<dim> &sim;
 
+      bool print_details;
```

Comment:
This variable has no documentation, but that is unfortunately the case also for everything below :-(